package.json

```diff
@@ -79,9 +79,6 @@
     "enzyme-adapter-react-16": "^1.1.1",
     "enzyme-to-json": "^3.2.2",
     "eslint": "^5.0.0",
-    "eslint-config-terra": "^2.2.0",
```

Comment:
😕 huh

Reply:
This was listed twice in the dependencies. Brett left a comment on the previous PR: #2141. It's still listed further down.

Reply:
I'd say we might as well keep the one with a higher minor version